workflow-bridge-lambda/src/index.ts

```diff
@@ -16,6 +16,7 @@ async function getPinboardByComposerId(composerId: string) {
   const contentResponse = await fetch(
     `${WORKFLOW_DATASTORE_API_URL}/content/${composerId}`
   );
+  // @ts-ignore
```

Comment:
what's being ignored, is there any way to get them fixed? it's a bit of a shame to have these warnings coming in

Reply:
factored these out (for posterity it was ignoring `TS2571: Object is of type 'unknown'` on the await beneath)